src/main/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfig.java

```diff
@@ -251,6 +254,26 @@ public class JdbcSinkConnectorConfig {
             .withDefault(false)
             .withDescription("Allowing to insert explicit value for identity column in table for SQLSERVER.");
 
+    public static final Field COLUMN_INCLUDE_LIST_FIELD = Field.create(COLUMN_INCLUDE_LIST)
```

Comment:
@Naros Since the description focuses about fields, should we rename the field name to `FIELD_INCLUDE_LIST` or could you suggest some alternative description?

Reply:
I think that makes sense @ani-sha.  This also aligns with the terminology with `primary.key.fields` as we don't use the term columns there either.